lib/core/src/lib/datatable/datatable.module.ts

```diff
@@ -73,7 +74,9 @@ import { AmountCellComponent } from './components/amount-cell/amount-cell.compon
         ResizableModule,
         DataColumnModule,
         BooleanCellComponent,
-        AmountCellComponent
+        DataColumnModule,
```

Comment:
I'm not sure but it look like the DataColumnModule is imported two times.

Reply:
nice catch, removed, thanks